parse.go

```diff
@@ -64,9 +64,20 @@ type command struct {
 	dest        path
 	specs       []*spec
 	subcommands []*command
+	groups      []*command
 	parent      *command
 }
 
+// getSpecs returns all specs of the command, plus all the option groups.
```

Comment:
maybe write this as 

`getSpecs gets all the specs from this command plus all nested option groups, recursively through descendants`

Reply:
Updated the text accordingly.